scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -374,32 +379,33 @@ trait ClassLikeSupport:
   def mkParameter(argument: ValDef,
     prefix: Symbol => String = _ => "",
     isExtendedSymbol: Boolean = false,
-    isGrouped: Boolean = false) =
+    isGrouped: Boolean = false,
+    memberInfo: Option[Map[String, TypeRepr]] = None) =
```

Comment:
It seems that `Map[String, TypeRepr]` with empty map as default should work here

Reply:
Will change